medmodels/data_synthesis/mtgan/modules/postprocessor.pyi

```diff
@@ -0,0 +1,59 @@
+from typing import Any, Dict, List, Literal, TypedDict
+
+import sparse
+import torch
+from torch import nn
+from typing_extensions import TypeAlias
+
+from medmodels.data_synthesis.mtgan.modules.preprocessor import PreprocessingAttributes
+from medmodels.medrecord.medrecord import MedRecord
+from medmodels.medrecord.types import MedRecordAttribute, NodeIndex
+
+AttributeType: TypeAlias = Literal["categorical", "continuous", "temporal"]
```

Comment:
Maybe we can use the AttributeType class from the Schema here? Since these are the same Types